scripts/e2e.ts

```diff
@@ -56,7 +56,10 @@ class E2ETestRunner extends events.EventEmitter {
 
   #testProcess: null | childProcess.ChildProcess = null;
   startTestProcess(): Promise<void> {
-    const args = process.argv.slice(2).filter(x => x !== '--serial');
+    const args = process.argv.slice(2).filter(x => x !== '--serial').map((s) => {
+      return s[0] === '-' ? s : s.replace(/\\/g, '/');
+    });
```

Comment:
Can you add a comment on why the replacement is necessary? And why "option arguments" are excluded from the transformation? What happens if an argument is `--file=e2e\foobar.baz`?

Reply:
This doesn't have to be a general solution, only one that handles the possible arguments that are used to run e2e tests. Which as far as I can tell are an optional list of filenames. But I'm leaving the `-*` alone because those arguments are intended for playwright internally.

From `node ./node_modules/@playwright/test/cli.js test --help` this could be an issue for `--config|-c FILE` or `--output DIR` but that's easily handled by using `/` or `\\` on the command-line. And it's not a typical use case for us.